docs/content/python/reference/lib.md

```diff
@@ -43,6 +43,6 @@ frameworks and ORMS.
 We just released the next version of our roles
 library.
 
-[Check it out here!](/guides/new-roles)
+[Check it out here!](/guides/roles)
```

Comment:
This callout is titled `Adding roles to your application with SQLAlchemy?` but we link to the non-SQLAlchemy roles page.

I think we should change the callout title